### PR TITLE
update movie tutorial for latest version

### PR DESCRIPTION
**Pull request by furkanevran**

### CODE REVIEW

File: tutorials/movies/adding_primary_and_gallery_images.md [hunk 1]
```diff
@@ -34,22 +34,21 @@ Then modify MovieRow.cs and PersonRow.cs:
...
         }
 
-        [DisplayName("Gallery Images"), 
+        [DisplayName("Gallery Images"),
          MultipleImageUploadEditor(FilenameFormat = "Person/GalleryImages/~")]
         public string GalleryImages
         {
-            get { return Fields.GalleryImages[this]; }
+            get => Fields.GalleryImages[this];
             set { Fields.GalleryImages[this] = value; }
```
Comment (volkanceylan) [inline tutorials/movies/adding_primary_and_gallery_images.md:52]: use arrow functions for both please

File: tutorials/movies/adding_a_movie_kind_field.md [hunk 2]
```diff
@@ -100,17 +94,16 @@ If we build and run our project now, we'll see that there is no change in the Mo
...
     // ...
     [FormScript("MovieDB.Movie")]
-    [BasedOnRow(typeof(Entities.MovieRow))]
+    [BasedOnRow(typeof(Entities.MovieRow), CheckNames = true)]
     public class MovieForm
     {
         // ...
-        public MovieKind Kind { get; set; }
         public Int32 Runtime { get; set; }
+        public MovieKind Kind { get; set; }
```
Comment (volkanceylan) [inline tutorials/movies/adding_a_movie_kind_field.md:106]: is there a reason for switching places of Runtime and Kind in form?

File: tutorials/movies/adding_primary_and_gallery_images.md [hunk 3]
```diff
@@ -70,30 +69,30 @@ namespace MovieTutorial.MovieDB.Entities
...
+            set => Fields.PrimaryImage[this] = value;
         }
 
-        [DisplayName("Gallery Images"), 
+        [DisplayName("Gallery Images"),
          MultipleImageUploadEditor(FilenameFormat = "Movie/GalleryImages/~")]
         public string GalleryImages
         {
-            get { return Fields.GalleryImages[this]; }
+            get => Fields.GalleryImages[this];
```
Comment (volkanceylan) [inline tutorials/movies/adding_primary_and_gallery_images.md:86]: arrow functions for both...

File: tutorials/movies/customizing_movie_interface.md [hunk 4]
```diff
@@ -262,9 +268,11 @@ InstanceName corresponds to singular name and is used in New Record (New Movie)
 namespace MovieTutorial.MovieDB.Entities
 {
     // ...
-    [ConnectionKey("Default"), DisplayName("Movies"), InstanceName("Movie"), 
-     TwoLevelCached]
-    public sealed class MovieRow : Row, IIdRow, INameRow
+    [ConnectionKey("Default"), Module("MovieDB"), TableName("[mov].[Movie]")]
+    [DisplayName("Movies"), InstanceName("Movie")]
+    [ReadPermission("Administration:General")]
```
Comment (volkanceylan) [inline tutorials/movies/customizing_movie_interface.md:273]: not sure why we use Administration:General for movie permissions here?

File: tutorials/movies/customizing_quick_search.md [hunk 5]
```diff
@@ -159,6 +161,7 @@ We can now use intellisense to replace hardcoded field names with compile time c
 namespace MovieTutorial.MovieDB
 {
     //...
+    import fld = MoviesRow.Fields
     public class MovieGrid extends EntityGrid<MovieRow, any>
```
Comment (volkanceylan) [inline tutorials/movies/customizing_quick_search.md:165]: one line space between import and class

File: tutorials/movies/customizing_quick_search.md [hunk 6]
```diff
@@ -186,14 +188,14 @@ What about field titles? It is not so critical as field names, but can be useful
 namespace MovieTutorial.MovieDB
 {
     //...
+    import fld = MoviesRow.Fields
```
Comment (volkanceylan) [inline tutorials/movies/customizing_quick_search.md:191]: add one line space after

File: tutorials/movies/filtering_with_multiple_genre_list.md [hunk 7]
```diff
@@ -179,6 +185,11 @@ Modify *MyListHandler* in *MovieRepository.cs* like below:
 ```cs
 private class MyListHandler : ListRequestHandler<MyRow, MovieListRequest>
 {
+    public MyListHandler(IRequestContext context)
+    : base(context)
```
Comment (volkanceylan) [inline tutorials/movies/filtering_with_multiple_genre_list.md:189]: missing tab spacing before : base

File: tutorials/movies/the_cast_and_characters_they_played.md [hunk 8]
```diff
@@ -135,43 +135,38 @@ namespace MovieTutorial.MovieDB.Entities
...
     {
-        //... remove QuickSearch from FirstName
-        [DisplayName("First Name"), Size(50), NotNull]
+        //... remove QuickSearch and NameProperty from FirstName
+        [DisplayName("Firstname"), Size(50), NotNull]
         public String Firstname
         {
-            get { return Fields.Firstname[this]; }
-            set { Fields.Firstname[this] = value; }
+            get => fields.Firstname[this];
```
Comment (volkanceylan) [inline tutorials/movies/the_cast_and_characters_they_played.md:142]: https://english.stackexchange.com/questions/78894/firstname-or-first-name
I think there should be a space in first name and last name (unlike surname) for display name attrs

**Review by volkanceylan — COMMENTED** [review]
generally looks good, please check comments

File: tutorials/movies/adding_a_movie_kind_field.md (hunk 2 above)
Comment (furkanevran) [reply tutorials/movies/adding_a_movie_kind_field.md:106]: I thought it made more sense as we are currently adding Kind on MovieRow and it's below Runtime in the properties of the class

File: tutorials/movies/customizing_movie_interface.md (hunk 4 above)
Comment (furkanevran) [reply tutorials/movies/customizing_movie_interface.md:273]: [Generating Code For Movie Table doc page](https://github.com/serenity-is/docs/blob/master/tutorials/movies/generating_code_for_movie_table.md) here it says to leave permission as *Administration:General*

**Review by volkanceylan — APPROVED** [review]

**Pull request merged**